src/Settings.ts

```diff
@@ -11,6 +11,7 @@ import IPushbulletManagerConfigJson from "./handlers/pushbulletManager/IPushbull
 import IPushoverManagerConfigJson from "./handlers/pushoverManager/IPushoverManagerConfigJson";
 import ISettingsConfigJson from "./types/ISettingsConfigJson";
 import ITelegramManagerConfigJson from "./handlers/telegramManager/ITelegramManagerConfigJson";
+import IConfiguration from "./types/IConfiguration";
```

Comment:
It took me a minute to figure out why you added this but it's clever. Nice way to handle two different files that could be in two different locations in a nice, seamless, way.